deployment/postgres-init-db/sql/scheduler/init.sql

```diff
@@ -2,5 +2,15 @@
 
```

Comment:
Just a note that @patkenneally PR changes the location of each service's SQL schema. Everything will be eventually moved to:
`<your-gradle-subproject>/sql/<table-name>
`

Reply:
@goetzrrGit This is now addressed 👍🏻